debian/control

```diff
@@ -9,6 +9,7 @@ Build-Depends: debhelper (>= 8),
                python3-all,
                python3-setuptools,
                python-nose, python3-nose, python-mock,
+               python-enum34 python3-enum34,
```

Comment:
i think there are other places in baseplate assuming that py3 will be 3.4+, so i think it's ok to continue that here.